Woof/Woof/Helper/AppStyle/String+IconName.swift

```diff
@@ -25,5 +25,7 @@ extension String {
         static let filledPerson = "person.fill"
         /// The icon used in places where picture of a house is required.
         static let house = "house"
+        /// The icon used to indicate the copy to clipboard.
```

Comment:
Should we keep the consistent style of doc comments:
```suggestion
        /// The icon used ...
```
?

Reply:
Fixed